app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AddProductNutritionFactsFragment.java

```diff
@@ -754,17 +758,20 @@ private boolean isDataPer100() {
 
     @OnClick(R.id.btn_add_a_nutrient)
     void addNutrient() {
+        String[] nutrients = getResources().getStringArray(R.array.nutrients_array);
+        // sort string by ignoring accents on letters. The sorted strings will keep their accents.
+        Arrays.sort(nutrients, Collator.getInstance(Locale.US));
```

Comment:
I bet you wanted to use StringComparator. :P

Reply:
Thanks for the review.
This PR is experimental with Collator. I deleted the Comparator

I pushed the changes with Collator to https://github.com/openfoodfacts/openfoodfacts-androidapp/pull/3089